### DISCUSSION TITLE
舊手機新妙用

### DISCUSSION THREAD

**Discussion by Clare0808** [1101-GS4719-C/Python-Community-GS4719-C#2028] [General]
https://tech.udn.com/tech/story/123151/8127456?from=down_3c_news

換新手機的時候，都會覺得舊手機就這樣丟掉很可惜，因為它只是有點不好用，而不是全然壞掉。
我個人覺得文章的第1、2點蠻實用的。網路上有些居家安全的App應該都挺不錯的，下載下來取代買新的監視器，感覺可以省不少錢。而當成音樂播放器也是很棒的點子，有時候手機裡的音樂下載太多，會佔據很多的空間。對部分熱愛音樂的人來說，有一支專門來存音樂的手機真的很棒！

**Comment by CtyceT** [comment]
個人家中有將舊手機作為網路基地台使用（文章第六點）。主要原因是已用了4年的舊手機的關機鍵壞掉了無法操控，因此現在只能讓它在電池徹底壞掉之前作為家中的網路熱點提供其他3C用品使用。

**Reply by gigitilto** [reply]
我也是這樣用我的舊手機的，但我的舊手機開熱點後網路速度會變得非常慢，而且電池健康度比較差的關係，需要邊充電邊開熱點，導致手機會嚴重發燙，感覺還不如用家裡的WiFi。

**Comment by ds0w0** [comment]
我同意你的看法，其實，我經常繼承家中長輩的舊手機，換個電池後依然可以用好幾年，真的CP值很高，完全是把它用到回本。我爸是把舊手機當作影片播放器，用來追劇或看電影，效果很好。而我媽則是把舊手機用來存相片，這樣就不會占用她新手機的空間，而且也很方便隨時翻看。再加上文章中提到的用法，像是當監視器或音樂播放器，舊手機真的有很多妙用。

**Comment by hunteryang0108** [comment]
謝謝你的分享，不過我換新手機通常都是因為舊手機已經不堪用了，你提供的那些作法可能就不是那麼適合我。不過我也提供一些不能使用的手機的處理方法：可以拿到家附近的超商回收兌換購物折抵金。我曾經把家裡全部的舊手機拿去兌換，成功換得了一些日常生活用品，是一個很實用的方法推薦給你。

**Comment by 112121py** [comment]
感謝分享～
這是個蠻不錯的想法，讓舊手機除了被轉手賣掉或丟掉外有更多的用途，但如果要進行像文章說的的應用，感覺對手機的版本或是軟硬體會有一定程度上的要求，不然像第一點拿來當家用監視器好了，首先就要達到基本的影像需求，有些畫質沒有那麼好、會卡頓等等，可能也是一些需要處理的問題
